### PR TITLE
Add podspec to support cocoapods projects

### PR DESCRIPTION
**Pull request by kevinresol**

### CODE REVIEW

**Comment by n1ru4l** [comment]
Duplicate of https://github.com/phuochau/react-native-thumbnail/pull/23

**Comment by kevinresol** [comment]
Not sure if it should be inside the ios folder. Most libraries I encountered put it in the root.
And I am using my PR in my project, and it works.

**Comment by n1ru4l** [comment]
@kevinresol That is why I moved it from the `ios` folder to the project root in my pull request  👀

**Comment by kevinresol** [comment]
ahh I see, I didn't notice the move action 👍. Shall we merge it ?

**Comment by n1ru4l** [comment]
@kevinresol I actually like your PR more, since you add all the metadata to the Podfile. If you can delete the old podspec file (which is located in the `ios` folder) I would prefer your pull request. 😊

Sadly I have no merge/publish rights for this repository and I also tried to contact the maintainer via email without success. I have a few open pull requests that I would like to get merged. At the meantime I am maintaining my own fork 😢.

File: RNThumbnail.podspec [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = "https://github.com/phuochau/react-native-thumbnail/#readme"
+  s.source         = { :git => 'https://github.com/phuochau/react-native-thumbnail.git' }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '7.0'
+
+  s.preserve_paths = 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'iOS/*.{h,m}'
```
Comment (n1ru4l) [inline RNThumbnail.podspec]: it is `ios` not `iOS`

**Pull request merged**